Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3-alpine3.10
```

How about something more flexible so that things do not go stale quickly while still allow building an image with any Python Docker image tag?

```suggestion
ARG PYTAG=latest
FROM python:${PYTAG}
```